### PR TITLE
[chore] use metadata.Type in activedirectorydsreceiver

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

**Review by mx-psi — COMMENTED** [review]
Does this need changes in the README?

**Review by mx-psi — APPROVED** [review]

**Comment by mx-psi** [comment]
needs a rebase

File: receiver/activedirectorydsreceiver/metadata.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-type: activedirectorydsreceiver
```
Comment (djaglowski) [inline receiver/activedirectorydsreceiver/metadata.yaml:1]: This would be a breaking change. Is it really necessary to add the underscores?

Comment (djaglowski) [reply receiver/activedirectorydsreceiver/metadata.yaml:1]: Nevermind, I see this is correctly the type used in the config.

**Review by djaglowski — APPROVED** [review]

**Pull request merged**